### DISCUSSION TITLE
Deploy private git that I doesn't own (I have acces to it)

### DISCUSSION THREAD

**Discussion by NicolasChusseau** [vercel/community#2209] [Help] [answered]
### Summary

Hello everyone! 
I have on a private git repository a site in React. I don't own this repository, I just have access to it. I want to deploy it (the git owner gave me the authorization) on vercel, but when I arrive at the moment where I can choose the git repository it does not appear (but the git repositories I own are visible). I can't import it by clicking on "Import Third-Party Git Repository", because the repository is private and vercel can't access it. I tried to create a team by inviting the git owner, but the repository still does not appear.

If you have any ideas on how to solve this problem, thank you in advance.

**Comment by amyegan** [comment, marked as answer]
Hi @NicolasChusseau! It sounds like you're trying to create the project on a Pro team, but please correct me if I got that wrong. If you have the owner on your team, try having them connect the repository instead of you. It sounds like this could be related to permissions on the git repo since you are not the owner

**Reply by NicolasChusseau** [reply]
Yes, it can solve my problem, but it means that it is not possible to deploy someone else's private project.

**Reply by amyegan** [reply]
Right, Vercel needs to be given access to the private repository. So you can deploy your own private repo or a repo that belongs to an organization that you belong to. But a repo owned by another individual would need to be connected by that person to grant Vercel access to the project. 

If you can fork the private repo, or move it to an organization, then you would have control to let Vercel access the repo.
